tests/linalg_test.py

```diff
@@ -2085,6 +2085,12 @@ def testMatrixTranspose(self, shape, dtype):
     self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker)
     self._CompileAndCheck(jnp_fun, args_maker)
 
+  @jtu.sample_product(
+    n=[0, 1, 5, 10, 20],
+    )
+  def testHilbert(self, n):
+    args_maker = lambda: [n]
+    self._CheckAgainstNumpy(osp.linalg.hilbert, jsp.linalg.hilbert, args_maker)
```

Comment:
We should run `_CompileAndCheck` as well. Since `n` is static, this means the functions should be partially-evaluated with `n`, and use `args_maker = lambda: []`

Reply:
Done